.ci_support/osx_64_blas_implgenericchannel_targetsconda-forge_mainnumpy2.0python3.12.____cpythonrcNone.yaml

```diff
@@ -52,10 +52,14 @@ python:
 - 3.12.* *_cpython
 pytorch:
 - '2.4'
+rc:
+- None
 target_platform:
 - osx-64
```

Comment:
Here is the macOS `x86_64` build for Python 3.12 with `rc` set to `None`

The corresponding [Azure CI build]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1137925&view=logs&jobId=c8b8640e-759f-5af3-65b5-809c078c71b1&j=899534ae-d486-52b7-7b1c-5c9a77433eb6&t=c641a15e-a70f-5a9e-1db4-6b8087a1df4f ) ran for ~5hrs

So AFAICT this is working as expected

Reply:
That's because the version here currently does not contain `"rc"`. Switch to the 2.6.0.rc3 builds, and you'll see (if I'm right) that everything will get skipped.

Reply:
Please see my comment above: https://github.com/conda-forge/pytorch-cpu-feedstock/pull/280#discussion_r1906676886